edsnlp/utils/span_getters.py

```diff
@@ -35,24 +35,24 @@
 ]
 
 
-def get_spans(doc, span_getter):
+def get_spans(doc, span_getter, deduplicate=True):
```

Comment:
Can we add a test for this new behavior ?

Reply:
Yes good call !